tests/e2e/utils/src/flows/with-rest-api.js

```diff
@@ -81,6 +82,24 @@ export const withRestApi = {
 		const repository = SimpleProduct.restRepository( client );
 		await deleteAllRepositoryObjects( repository );
 	},
+	/**
+	 * Use api package to delete all orders.
+	 *
+	 * @return {Promise} Promise resolving once orders have been deleted.
+	 */
+	deleteAllOrders: async () => {
```

Comment:
`deleteAllRepositoryObjects()` could be updated to accept a status array as a parameter. That would eliminate duplicating the logic.

Reply:
I prefer that. I updated the delete to use this:

```javascript
deleteAllOrders: async () => {
	// We need to specfically filter on order status here to make sure we catch all orders to delete.
	const orderStatuses = ['pending', 'processing', 'on-hold', 'completed', 'cancelled', 'refunded', 'failed', 'trash'];
	const repository = Order.restRepository( client );
	await deleteAllRepositoryObjects( repository, null, orderStatuses );
},
``

And have the following set of default statuses being passed in:

```javascript
const deleteAllRepositoryObjects = async ( repository, defaultObjectId = null, statuses = [ 'draft', 'publish', 'trash' ] ) => {}
```